modules/kms-keyring-browser/test/kms_mrk_discovery_keyring_browser.test.ts

```diff
@@ -16,7 +16,6 @@ import {
   AlgorithmSuiteIdentifier,
   WebCryptoDecryptionMaterial,
 } from '@aws-crypto/material-management-browser'
-import { KMS as V2KMS } from 'aws-sdk'
```

Comment:
Same comment as above. Why does changing the default remove support? Or the need to test that support?